src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -1,6 +1,9 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
+parameters:
+    sylius_api.authorization_header: Authorization
```

Comment:
Maybe we could add the possibility to configure it by env variable?

Reply:
Good idea 👍